partials/footer-content.php

```diff
@@ -1,79 +1,160 @@
+                <h4>Legal</h4>
+                <ul>
+                    <li><a href="/legal/">Terms of use</a></li>
+                    <li><a href="/legal/privacy.htm">Privacy policy</a></li>
+                    <li><a href="/legal/cookies.htm">Cookies</a></li>
+                    <li><a href="/about/freedom-of-information/">Freedom of Information</a></li>
+                    <li><a href="/about/our-role/transparency/">Transparency</a></li>
+                    <li><a href="/legal/our-fees.htm">Our fees</a></li>
+                </ul>
+            </div>
+        </div>
     </div>
-    <div class="footer-col col-xs-12 col-sm-6 col-md-3">
-        <h4>Follow us</h4>
-        <ul>
-            <li><a href="https://twitter.com/@UKNatArchives" target="_blank">Twitter</a></li>
-            <li><a href="https://www.youtube.com/c/TheNationalArchivesUK" target="_blank">YouTube</a></li>
-            <li><a href="https://www.flickr.com/photos/nationalarchives" target="_blank">Flickr</a></li>
-            <li><a href="https://www.facebook.com/TheNationalArchives?_rdr=p" target="_blank" >Facebook</a></li>
-            <li><a href="https://commons.wikimedia.org/wiki/Commons:The_National_Archives_(United_Kingdom)" target="_blank">Wikimedia Commons</a></li>
-            <li><a href="/rss/">RSS</a></li>
-        </ul>
+    <div class="footer-col col-xs-12 col-sm-12 col-md-3">
+        <div class="row">
+            <div class="footer-col col-xs-12 col-sm-6 col-md-12">
+                <h4>Follow us</h4>
+                <div class="follow-us">
+                    <a href="https://twitter.com/@UKNatArchives" title="Follow us on Twitter" target="_blank">
+                        <img src="<?php echo get_stylesheet_directory_uri() ?>/img/social/twitter.png"
+                             alt="Twitter logo"></a>
+                    <a href="https://www.youtube.com/c/TheNationalArchivesUK" title="Follow us on YouTube"
+                       target="_blank">
+                        <img src="<?php echo get_stylesheet_directory_uri() ?>/img/social/youtube-play.png"
+                             alt="YouTube logo"></a>
+                    <a href="https://www.flickr.com/photos/nationalarchives" title="Follow us on Flickr"
+                       target="_blank">
+                        <img src="<?php echo get_stylesheet_directory_uri() ?>/img/social/flickr.png" alt="Flickr logo"></a>
+                    <a href="https://www.facebook.com/TheNationalArchives" title="Follow us on Facebook"
+                       target="_blank">
+                        <img src="<?php echo get_stylesheet_directory_uri() ?>/img/social/facebook.png"
+                             alt="Facebook logo"></a>
+                    <a href="http://www.nationalarchives.gov.uk/rss/" title="Follow us via RSS">
+                        <img src="<?php echo get_stylesheet_directory_uri() ?>/img/social/rss.png" alt="RSS logo"></a>
```

Comment:
Again same thing `<?php echo get_template_directory_uri(); ?>` instead of `<?php echo get_stylesheet_directory_uri() ?>`